docs/source/configuration/cors.mdx

```diff
@@ -1,146 +1,76 @@
 ---
 title: Configuring CORS in the Apollo Router
 sidebar_title: CORS
-description: Control access to your server's resources
+description: Control browser access to your router
 ---
 
-import { Link } from 'gatsby';
+> **This article describes CORS configuration that's specific to the Apollo Router**. For a more general introduction to CORS and common considerations, see the following sections:
+>
+> * [Why use CORS?](/apollo-server/security/cors#why-use-cors)
+> * [Choosing CORS options for your project](/apollo-server/security/cors#choosing-cors-options-for-your-project)
 
-[Cross-Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) (CORS) is an HTTP-header-based protocol that enables a server to dictate which origins can access its resources. Put another way, your server can specify which websites can tell a user's browser to talk to your server, and precisely which types of HTTP requests are allowed.
+By default, the Apollo Router enables _only_ Apollo Studio to initiate browser connections to it. If your supergraph serves data to other browser-based applications, you need to do one of the following in the `server.cors` section of your router's [YAML config file](./overview/#yaml-config-file):
 
-Apollo Router's default CORS behavior enables only the Apollo studio to tell a user's browser to connect to your server.
+* Add the origins of those web applications to the router's list of allowed `origins`.
+    * Use this option if there is a known, finite list of web applications that consume your supergraph.
+    * You _must_ use this option if clients need to [authenticate their requests with cookies](#passing-credentials-with-cors).
+* Enable the `allow_any_origin` option.
+    * Use this option if your supergraph is a public API with arbitrarily many web app consumers.
+    * With this option enabled, the router sends the [wildcard (`*`)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin#directives) value for the `Access-Control-Allow-Origin` header. This enables _any_ website to initiate browser connections to it (but they can't provide cookies or other credentials).
 
-If your Apollo Router is serving data for web applications, you want to either add it to the list of allowed origins, or switch to `allow_any_origin`.
+The following snippet includes an example of each option (use only one):
 
-> ⚠️ If your app is only visible on a private network and uses network separation for security, the default CORS behavior is **not secure**. See [Specifying origins](#specifying-origins) for more information.
```

Comment:
This article currently borrows heavily from the Apollo Server CORS article with few to no edits. To reduce content duplication as much as possible, I instead link to the relevant portions of that article and keep the focus here on the router as much as possible